dotnet/src/Skills/Skills.Core/ConversationSummarySkill.cs

```diff
@@ -37,6 +37,7 @@ public ConversationSummarySkill(IKernel kernel)
     {
         this._summarizeConversationFunction = kernel.CreateSemanticFunction(
             SemanticFunctionConstants.SummarizeConversationDefinition,
+            functionName: nameof(SummarizeConversationAsync),
```

Comment:
In our native functions, we have code that grabs the method name, but also strips off the "Async" suffix, as the AI shouldn't know or care about that little detail... I'm thinking, instead of trying to do that automatically here, could you add string constants for each of these semantic skill definitions in the same file as the prompts themselves:
https://github.com/microsoft/semantic-kernel/blob/main/dotnet/src/Skills/Skills.Core/SemanticFunctionConstants.cs

...where SummarizeConversationAsync is stored as "SummarizeConversation", etc?